src/legacy/js/functions/_externalLinkAccordionSection.js

```diff
@@ -6,9 +6,21 @@
  * @param idField - HTML id for the template
  */
 
+function getExternalLinkAccordionSectionLabel(idField) {
+    switch(idField) {
+        case "filterable-dataset":
+            return "Try a new way to download data"
+        case "link":
+            return "You might also be interested in"
+        default:
+          return `Related ${idField}`
```

Comment:
Is there potential for the `idField` to be empty?

Reply:
No. Well, very _very_ unlikely and worst case scenario it would return `Related undefined` rather than bork.